settings/provisioner/topology/loadbalancer.yaml

```diff
@@ -0,0 +1,53 @@
+name: "loadbalancer"
+cpu: "2"
+memory: "4096"
+type: "external"
```

Comment:
why is this needed here?
the definition for external LB is osp-d based no?
Haveing a dedicated VM for it already config it as such.